### PR TITLE
VEN-383 | Customer Details on customers list page

### PR DESCRIPTION
**Pull request by Nizar-Rahme**

### CODE REVIEW

File: apollo.config.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 // eslint-disable-next-line
-require('dotenv').config({ path: '.env.development.local' });
```
Comment (Zeukkari) [inline apollo.config.js]: Can we revert this or is this relevant to the PR?

Comment (Nizar-Rahme) [reply apollo.config.js]: It was relevant to this task since it configures Apollo extension to read the API URI  from `.env` file which activates the query auto-correction in the code editor.

Comment (Zeukkari) [reply apollo.config.js]: I don't quite understand why this should be necessary. I tried reverting this and now I'm getting a confusing error when running code generation.

```
╰─ yarn codegen                                                                                                                                                              ─╯
yarn run v1.21.1
$ yarn schema:download && yarn types:generate
$ apollo service:download --endpoint=https://venepaikka-federation.test.kuva.hel.ninja/
 ›   Warning: apollo update available from 2.21.1 to 2.21.2.
  ✔ Loading Apollo Project
  ✖ Saving schema to schema.json
    → spurious results.
Error: Cannot use GraphQLSchema "[object GraphQLSchema]" from another module or realm.
Ensure that there is only one instance of "graphql" in the node_modules
directory. If different versions of "graphql" are the dependencies of other
relied on modules, use "resolutions" to ensure only one version is installed.
https://yarnpkg.com/en/docs/selective-version-resolutions
Duplicate "graphql" modules cannot be used at the same time since different
versions may have different capabilities and behavior. The data from one
version used in the function from another could produce confusing and
spurious results.
    at instanceOf (~/workspace/berth-reservations-admin/node_modules/apollo/node_modules/graphql/jsutils/instanceOf.js:28:13)
    at isSchema (~/workspace/berth-reservations-admin/node_modules/apollo/node_modules/graphql/type/schema.js:36:34)
    at assertSchema (~/workspace/berth-reservations-admin/node_modules/apollo/node_modules/graphql/type/schema.js:40:8)
    at validateSchema (~/workspace/berth-reservations-admin/node_modules/apollo/node_modules/graphql/type/validate.js:44:28)
    at assertValidSchema (~/workspace/berth-reservations-admin/node_modules/apollo/node_modules/graphql/type/validate.js:68:16)
    at assertValidExecutionArguments (~/workspace/berth-reservations-admin/node_modules/apollo/node_modules/graphql/execution/execute.js:136:35)
    at executeImpl (~/workspace/berth-reservations-admin/node_modules/apollo/node_modules/graphql/execution/execute.js:86:3)
    at execute (~/workspace/berth-reservations-admin/node_modules/apollo/node_modules/graphql/execution/execute.js:64:63)
    at Object.introspectionFromSchema (~/workspace/berth-reservations-admin/node_modules/apollo/node_modules/graphql/utilities/introspectionFromSchema.js:31:37)
    at Task.task (~/workspace/berth-reservations-admin/node_modules/apollo/lib/commands/service/download.js:21:82)
error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

Comment (Zeukkari) [reply apollo.config.js]: https://github.com/City-of-Helsinki/berth-reservations-admin/pull/102/ Can we revert this apollo config change? Other than that this looks good.
